addons/web/static/src/core/dropdown/accordion_item.js

```diff
@@ -0,0 +1,30 @@
+/** @odoo-module **/
+
+import { Component, onPatched, useState } from "@odoo/owl";
+
+export const ACCORDION = Symbol("Accordion");
+export class AccordionItem extends Component {
+    setup() {
+        super.setup();
+        this.state = useState({
+            open: false,
+        });
+        this.parentComponent = this.env[ACCORDION];
+        onPatched(() => {
+            this.parentComponent?.accordionStateChanged?.();
+        });
+    }
+
+    static template = "web.AccordionItem";
+    static components = {};
+    static props = {
+        slots: {
+            type: Object,
+            shape: {
+                default: {},
+            },
+        },
+        description: String,
+        selected: Boolean,
+    };
```

Comment:
Nitpicking about coding style:
not sure if we want to enforce this practice, but generally we place static members of a class above the rest of its members.